cmd/benchmarks/run-benchmarks.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/evergreen-ci/timber"
+)
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	fmt.Println("running basic sender benchmark...")
+	if err := timber.RunBasicSenderBenchmark(ctx); err != nil {
```

Comment:
does it make sense to have the benchmark functions public in the timber package.

Reply:
I think so, the benchmarks I wrote require access to private functions in the timber package